src/Request/StreamExtensions.cs

```diff
@@ -7,7 +7,12 @@
 
     public static class StreamExtensions
     {
-
+        /// <summary>
+        /// Gets the RequestBody Stream as string in the specified <see cref="Encoding"/> (optional)
+        /// </summary>
+        /// <param name="stream">Current <see cref="Stream"/></param>
+        /// <param name="encoding">The character encoding to use or <see cref="Encoding.UTF8"/> by default</param>
+        /// <returns>Current string content of the RequestBody Stream</returns>
```

Comment:
Current <see cref="String"/> content of the <see cref="HttpRequest" /> Body <see cref="Stream"/>